packages/cssnano-preset-advanced/README.md

```diff
@@ -122,7 +122,13 @@ This plugin is loaded with the following configuration:
 
 > Sorts CSS declarations fast and automatically in a certain order.
 
-This plugin is loaded with its default configuration.
+This plugin is loaded with the following configuration:
+
+```js
+{
+	customOrder: null
```

Comment:
Don't think what we need this option, no one will constantly keep their list, it's too time consuming and takes a lot of time and effort, let's remove it from here

Reply:
What do you mean? It's there so that the advanced preset does full sorting resulting in more optimization. Note that this is generated documentation too.